ani.voskanyan/repo/Lesson_7/exercise_04/exercise_04.py

```diff
@@ -0,0 +1,15 @@
+#!usr/bin/env python3
+
+try:
+    string_input = input("Please enter your text: ")
+    width = int(input("Please enter the width of the terminal: "))
+except ValueError:
+        print("The width of the terminal can be only an integer. Try again.")
+        width = int(input("Please enter the width of the terminal: "))

```

Comment:
LW: If the user inputs the wrong input a second time it will print an error message from the python. Use loop if you want to prevent unwanted input.